app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title><%= t "title" %></title>
+    <%= csrf_meta_tags %>
```

Comment:
Dòng này có tác dụng gì

Reply:
csrf_meta_tag chèn những gì thực chất là chữ ký số vào trang, đóng vai trò xác minh rằng các yêu cầu đến máy chủ ứng dụng, trên thực tế, từ người dùng đã đăng nhập đúng cách. Điều này giúp ngăn việc tạo kịch bản chéo trang (một tập lệnh trên một trang hoàn toàn không liên quan đến yêu cầu tắt, GMail, trong khi bạn đã đăng nhập vào Gmail của mình trong một tab khác).